PyRDF/Node.py

```diff
@@ -79,4 +61,11 @@ def _graph_prune(self):
             
             return False
 
+        elif self.operation and not self.next_nodes and (self.operation.op_type == Operation.Types.TRANSFORMATION):
```

Comment:
If would add a couple of methods to `Operation.py` so that this `self.operation.op_type == Operation.Types.TRANSFORMATION` becomes this `self.operation.is_transformation()` (same for actions)

Reply:
👍